### ISSUE TITLE
Writes using `RETURNING` must use "Query" method

### ISSUE THREAD

**Issue by RangelReale** [bxcodec/dbresolver#28]
We use some INSERT/UPDATE commands that have a RETURNING clause, so we need to use `.Query` even if we are writing. 

Wouldn't the library use the wrong db connection in this case?

**Comment by Nasfame** [comment]
@RangelReale insert/update operation is a write operation i.e this query goes to a primary db/ write db. Therefore the returning clause should return correctly.



@RangelReale  can u post the sample code ?

**Comment by RangelReale** [comment]
the documentation says

```
Replica Databases will be used when you call these functions
* Query
```

do you parse the SQL on the "Query" method to see if it is a write operation (with possibly RETURNING)?

**Comment by Nasfame** [comment]
I get your point. Now i see your point. @RangelReale Thanks for pointing it out. We will look into it.

@bxcodec this is a serious bug.

@bxcodec  we have to ensure that queries with "Returning" clause goes to primary dbs. 

@RangelReale can u post some of the sample queries that we can add to our testsuite..

**Comment by Nasfame** [comment]
Fix Ready for #33
